en/components/grid/grid.md

```diff
@@ -616,6 +616,8 @@ public ngOnInit() {
 |Grid `OnPush` ChangeDetectionStrategy |The grid operates with `ChangeDetectionStrategy.OnPush` so whenever some customization appears make sure that the grid is notified about the changes that happens.|
 | Columns have a minimum allowed column width. Depending on the [`displayDensity`]({environment:angularApiUrl}/classes/igxgridcomponent.html#displaydensity) option, they are as follows: <br/>"compact": 24px <br/> "cosy": 32px <br/> "comfortable ": 48px | If width less than the minimum allowed is set it will not affect the rendered elements. They will render with the minimum allowed width for the corresponding [`displayDensity`]({environment:angularApiUrl}/classes/igxgridcomponent.html#displaydensity). This may lead to an unexpected behavior with horizontal virtualization and is therefore not supported.
 | Row height is not affected by the height of cells that are not currently rendered in view. | Because of virtualization a column with a custom template (that changes the cell height) that is not in the view will not affect the row height. The row height will be affected only while the related column is scrolled in the view.
+| Scroll position of components that use igxForOf is not preserved |When changing the context of an element, that uses igxForOf, the scrollbar position would not be preserved, even though the data position would be. That inconsistency occurs when the content is created and after that the DOM nodes get changed. In such a case, the nodes are getting detached from the DOM and then reattached again in a different place. That's why they will not keep their scroll state, but ever since the component is not destroyed/recreated, it will still have its internal state. Some possible workarounds are: <br/> <ul><li>Resetting the DOM node's state, for example by wrapping it in an ngIf, that'd follow it's state (the tab gets selected => attaching nodes to the DOM tree, the tab gets disselected => removing nodes from the tree).</li><li>Persisting the state, i.e. determining whether the element has been re-added to the DOM tree and resetting its scroll position manually.</li></ul>
```

Comment:
I think the description should be `igxGrid` specific. Here's what I suggest:
|`igxGrid` won't preserve its scroll position if put in projected container like `<ng-content>`. | This is the case with `igxGrid` in `igxTabs`. When tabs get changed the grid won't preserve its scrollbar position, altough the grid content is at correct offset in the data source. The reason is that `igxGrid` uses `igxForOf` directive internally, which cannot identify the detaching and re-attaching of the DOM elements from the DOM tree and thus cannot update the scrollbar thumb position.|

Reply:
Done